monai/bundle/scripts.py

```diff
@@ -415,12 +419,22 @@ def ckpt_export(
     # convert to TorchScript model and save with meta data, config content
     net = convert_to_torchscript(model=net)
 
+    extra_files: Dict = {}
+    for i in ensure_tuple(config_file_):
+        # split the filename and directory
+        _, filename = os.path.split(i)
```

Comment:
```suggestion
        filename = os.path.basename(i)
```

Reply:
Sure, updated in the latest commit.

Thanks.